content/lessons/chapter-3/coop-2.tsx

```diff
@@ -166,7 +166,7 @@ export default function Coop2({ lang }) {
   }, [])
 
   return (
-    <div className="fade-in flex flex-col items-center gap-[20px] px-[15px] py-[30px] md:px-[75px] md:pt-[40px]">
+    <div className="fade-in flex flex-col items-center  gap-[20px] px-[15px] py-[30px] md:pt-[40px] 2xl:gap-[30px] 2xl:px-[75px]">
```

Comment:
Try to refrain from using arbitrary pixel values as much as possible, `pt-[40px]` could instead be `pt-10` instead, and unless absolutely necessary rather than doing `py-[30px]` use `py-8`.

Reply:
oh okay but py-8 is 32px is that fine?